test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -776,6 +812,25 @@ public static TheoryData<CreateTokenTheoryData> CreateJWSTheoryData
                     new CreateTokenTheoryData
                     {
                         First = true,
+                        TestId = "ValidUsingTokenType",
+                        Payload = Default.PayloadString,
```

Comment:
Same here, Payload seems to be irrelevant.

Reply:
Fixed it.